### PR TITLE
MWEB-3035 find adinsertion confirmed events by flag spt_is_adinsertio…

### PR DESCRIPTION
**Pull request by dprinczkel**
…n_confirmation

### CODE REVIEW

File: src/SPT-Pulse/926_SPT-general.js [hunk 1]
```diff
@@ -20,12 +20,12 @@ willhabenSPT = {
         AD_TYPE_ID: "ad_type_id",
         EVENT_NAME: "event_name",
         SPT_CUSTOM: "spt_custom",
-        VERTICAL_ID: "vertical_id",
+        VERTICAL_ID: "vertical_id"
```
Comment (fabb) [inline src/SPT-Pulse/926_SPT-general.js]: don't you have `npm install`ed? prettier should auto-format that consistently with the pre-commit hook

File: src/SPT-Pulse/926_SPT-general.js [hunk 2]
```diff
@@ -156,23 +156,26 @@ willhabenSPT = {
...
-                case willhabenSPT.EVENTS.AD_INSERTION_CONFIRMED_MWEB:
-                case willhabenSPT.EVENTS.AD_INSERTION_PAID_CONFIRMED_MWEB:
-                    b["spt_publisher_id"] = b.seller_uuid || " ";
-                    if (b["is_private"]) {
-                        b["spt_publisher_type"] = b["is_private"] === "true" ? willhabenSPT.PUBLISHER_TYPE_ENUM.PRIVATE : willhabenSPT.PUBLISHER_TYPE_ENUM.PRO;
-                    }
+                    setPublisher();
                     break;
             }
+            if (b["spt_is_adinsertion_confirmation"] === "true") { //extension "SPT AI confirmation" should be executed before
```
Comment (fabb) [inline src/SPT-Pulse/926_SPT-general.js]: could the content of the `SPT AI confirmation` extension just be added to be part of this file?

File: src/SPT-Pulse/926_SPT-general.js (hunk 1 above)
Comment (dprinczkel) [reply src/SPT-Pulse/926_SPT-general.js]: thanks for the tip, added it

File: src/SPT-Pulse/926_SPT-general.js (hunk 2 above)
Comment (dprinczkel) [reply src/SPT-Pulse/926_SPT-general.js]: yes good idea

**Pull request merged**